src/test/run-pass-fulldeps/proc-macro/span-api-tests.rs

```diff
@@ -12,6 +12,7 @@
 // aux-build:span-test-macros.rs
 
 // ignore-pretty
+// ignore-stage1
```

Comment:
why is that?

Reply:
why are you changing a test that should be unaffected? Is this only for the snapshot? In that case, I think you should add a `SNAP: remove after snapshot` comment.

Reply:
I've been testing the fmt::Arguments changes on stage 1, but it keeps failing on this test even without my changes. Someone forgot to add the flag, which is required for most proc-macro tests.

Reply:
I don't think it has anything to do with snapshots, just with compiler plugins. I can remove this comment from this PR. It's just an annoyance I encountered

Reply:
removed and squashed